lib/build.command.js

```diff
@@ -20,15 +20,13 @@ module.exports = function (Aquifer) {
   .action((options) => {
     let path = require('path');
     let settings = Aquifer.project.config;
-    let makeFile = path.join(Aquifer.projectDir, settings.paths.make);
     let copy = options.copy === true;
-    let directory = options.directory || settings.paths.build;
+    let directory = options.directory || settings.build.directory;
     let build = new Aquifer.api.build(Aquifer);
 
     // Create build.
     build.create(directory, {
-      symlink: !copy,
-      makeFile: makeFile
+      symlink: !copy
```

Comment:
Yoda-like?More detail please if there is a problem as its not apparent the expected results from your review here.